bftengine/src/preprocessor/RequestProcessingState.hpp

```diff
@@ -96,6 +96,8 @@ class RequestProcessingState {
   const std::string cid_;
   const ReqId reqSeqNum_;
   const uint64_t entryTime_;
+  const std::string clientRequestSignature_;
+  const uint32_t clientRequestSignatureLen_;
```

Comment:
clientRequestSignature_ is a string now, why do you need clientRequestSignatureLen_?

Reply:
I was first thinking to have it intentionally explicit since all other messages hold the same sig/siglen.
Rethinking, I'll remove that..